integration_tests/tests/apply/dashboard.cy.ts

```diff
@@ -65,7 +65,52 @@ context('All applications', () => {
   })
 
   it('supports sorting by createdAt', () => {
-    shouldSortByField('createdAt')
+    // Given I am logged in
+    cy.signIn()
+
+    // And there is a page of applications
+    const applications = applicationSummaryFactory.buildList(10)
+
+    cy.task('stubAllApplications', { applications, page: '1' })
+    cy.task('stubAllApplications', {
+      applications,
+      page: '1',
+      sortBy: 'createdAt',
+      sortDirection: 'asc',
+    })
+
+    // When I access the applications dashboard
+    const page = DashboardPage.visit(applications)
+
+    // Then I should see the first result
+    page.shouldShowApplications()
+
+    // Then the API should have received a request for the sort
+    cy.task('verifyDashboardRequest', { page: '1', sortBy: 'createdAt', sortDirection: 'desc' }).then(requests => {
+      expect(requests).to.have.length.greaterThan(0)
+    })
+
+    // When I sort by created at
+    page.clickSortBy('createdAt')
+
+    // Then the API should have received a request for the sort
+    cy.task('verifyDashboardRequest', { page: '1', sortBy: 'createdAt', sortDirection: 'desc' }).then(requests => {
```

Comment:
This assertion is the same as the one on line 89. It would be good to assert the number of requests that we expect

Reply:
sortby is falsy then we have in applicationService we have sortBy cretaedAt by default

Reply:
yes we can do I assert the number of requests but it will be cleaner to understand on first request and after clicking sort to check so easily i can undestand

Reply:
Under what circumstances could the assertion on line 89 pass but the one on 97 fail?